lib/config.js

```diff
@@ -18,3 +18,13 @@ config.profile.kms.ipAllowList = [];
 config.profile.kms.defaultKmsModule = 'ssm-v1';
 // ensure default KMS module is overridden in deployments
 config.ensureConfigOverride.fields.push('profile.kms.defaultKmsModule');
+
+config.profile.profileAgent = {
+  // One month time threshold (in milliseconds) for triggering zcap refresh.
+  zcapRefreshThreshold: 30 * 24 * 60 * 60 * 1000,
+  // TTL for profile agent's EDV zcaps: 365 days
+  defaultZcapTtl: 365 * 24 * 60 * 60 * 1000,
```

Comment:
```suggestion
  edvZcapTtl: 365 * 24 * 60 * 60 * 1000,
```

Reply:
Renamed `defaultZcapTtl` and updated the comments [b2cd058](https://github.com/digitalbazaar/bedrock-profile/pull/99/commits/b2cd0589398a82537e8b9bf61e6e084954bb6851)